<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ATMTakeMoneyForm</class>
 <widget class="QWidget" name="ATMTakeMoneyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>161</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertical_layout_a">
    <item>
     <widget class="QPushButton" name="button_100">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>100</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_200">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>200</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_400">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>400</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>90</y>
     <width>160</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertical_layout_b">
    <item>
     <widget class="QPushButton" name="button_800">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>800</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_1000">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1,000</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_2000">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2,000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_return">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>221</x>
     <y>90</y>
     <width>251</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertical_layout_c">
    <item>
     <widget class="QLabel" name="label_money">
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>请选择或输入取款金额</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="line_edit_money"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="text_browser_info">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>190</y>
     <width>256</width>
     <height>111</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;本机目前只提供100纸币，数目必须为100元的整数倍，每日单笔取款限额为2,000元。&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line_a">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>16</width>
     <height>271</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_b">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>90</y>
     <width>16</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontal_layout_a">
    <item>
     <widget class="QLabel" name="label_timer">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>倒计时：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLCDNumber" name="lcd_timer">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="button_accept">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>330</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
